### DISCUSSION TITLE
How to run starboard on an air-gapped envrionment?

### DISCUSSION THREAD

**Discussion by strus38** [aquasecurity/starboard#612] [Questions & Help] [answered]
all is in the question... of course I can pre-pull the images in a private registry, bu for the CVE database, how this would work?
I know that trivy supports air-gapped environment, but I didi not see the parameters I can set during starboard install.
Thanks

**Comment by deegee-dev** [comment]
Apologies, I did not intend for this to be a suggested answer but just to add on to this, I'm aware of #541. 
Is it possible to add on 
> --skip-update

to the list of supported argument for trivy?

**Reply by strus38** [reply]
Where in the starboard values of Helm?
trivy:
  imageRef: admin/trivy:0.16.0
  mode: Standalone
  serverURL:
  httpProxy:
  httpsProxy:
  noProxy:
  severity: UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL

**Comment by danielpacak** [comment, marked as answer]
Looks like duplicate of #613 and #617
